src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -86,22 +86,22 @@ function IOURequestStepConfirmation({
             return translate('iou.categorize');
         }
         if (isRequestingFromTrackExpense) {
-            return translate('iou.request');
+            return translate('iou.submitExpense');
```

Comment:
I mentioned this already, but this is changing the copy so just making sure this was done on purpose 😄